app/views/faqs/edit.html.erb

```diff
@@ -0,0 +1,3 @@
+<% content_for :title, "Editing faq" %>
+<h1 class="mb-3 text-center">Editing FAQ</h1>
+<%= render "form", faq: @faq %>
```

Comment:
We can use the same -form partial for New and Edit. In Edit we need to pass @fac variable to the form. Please notice, that I removed a title from the submit button in the _form.html.erb. Rails knows when to make it "Create' and when 'Update'